installer/upgrader/Migrations/V5/leaveLangString.yaml

```
@@ -0,0 +1,287 @@
+  -
+    value: 'Multiple Employees'
+    unitId: multiple_employees
+  -
+    value: '{empMatchCount,plural, =0{Matches no employee} one{Matches (1) employee} other {matches (#) employees}}'
+    unitId: matches_emp_count_employees
+  -
+    value: 'No leave types defined'
+    unitId: no_leave_types_defined
+  -
+    value: 'Should be a number with upto 2 decimal places'
+    unitId: should_be_a_number_with_2_decimal_places
+  -
+    value: 'Edit Leave Entitlement'
+    unitId: edit_leave_entitlement
+  -
+    value: 'Entitlement'
+    unitId: entitlement
+  -
+    value: 'Used amount exceeds the current amount'
+    unitId: used_amount_exceeds_the_current_amount
+  -
+    value: 'Leave Entitlements'
+    unitId: leave_entitlements
+  -
+    value: 'My Leave Entitlements'
+    unitId: my_leave_entitlements
+  -
+    value: 'Is Entitlement Situational?'
+    unitId: is_entitlement_situational
+  -
+    value: 'Add Leave Type'
+    unitId: add_leave_type
+  -
+    value: 'Edit Leave Type'
+    unitId: edit_leave_type
+  -
+    value: 'Leave Types'
+    unitId: leave_types
+  -
+    value: 'Leave Entitlements and Usage Report'
+    unitId: leave_entitlement_and_usage_report
+  -
+    value: 'Generate For'
+    unitId: generate_for
+  -
+    value: 'Leave Type'
+    unitId: leave_type
+  -
+    value: 'Include Past Employees'
+    unitId: include_past_employees
+  -
+    value: 'My Leave Entitlements and Usage Report'
+    unitId: my_leave_entitlement_and_usage_report
+  -
+    value: 'Apply Leave'
+    unitId: apply_leave
+  -
+    value: 'No Leave Types with Leave Balance'
+    unitId: no_leave_types_with_leave_balance
+  -
+    value: 'Partial Days'
+    unitId: partial_days
+  -
+    value: 'Assign Leave'
+    unitId: assign_leave
+  -
+    value: 'Start Day'
+    unitId: start_day
+  -
+    value: 'End Day'
+    unitId: end_day
+  -
+    value: 'Assign'
+    unitId: assign
+  -
+    value: 'Leave List'
+    unitId: leave_list
+  -
+    value: 'Show Leave with Status'
+    unitId: show_leave_with_status
+  -
+    value: 'My Leave Request Details'
+    unitId: my_leave_request_details
+  -
+    value: 'Leave Request Details'
+    unitId: leave_request_details
+  -
+    value: 'My Leave List'
+    unitId: my_leave_list
+  -
+    value: 'Show Leave with Status'
+    unitId: show_leave_with_status
+  -
+    value: 'Updating Entitlement'
+    unitId: updating_entitlement
+  -
+    value: 'Matching Employees'
+    unitId: matching_employees
+  -
+    value: 'The selected leave entitlement will be applied to the following employees.'
+    unitId: selected_leave_entitlement_applied_to_following_employees
+  -
+    value: 'Old Entitlement'
+    unitId: old_entitlement
+  -
+    value: 'New Entitlement'
+    unitId: new_entitlement
+  -
+    value: 'Situational Leave'
+    unitId: situational_leave
+  -
+    value: 'These leave will be excluded from reports unless there''s some activity. 
+    E.g. maternity leave, jury duty leave.'
```

can't we go with single line?
```suggestion
    value: "These leave will be excluded from reports unless there's some activity. 
    E.g. maternity leave, jury duty leave."
```

---

changed